packages/engine/src/proto.rs

```diff
@@ -379,7 +379,7 @@ pub struct ProcessedExperimentRun {
 }
 
 // TODO: Replace with UUID?
-pub type ExperimentId = String;
+pub type ExperimentRunId = String;
```

Comment:
Maybe we want it to name `ExperimentName`? This would avoid further confusion in the code and is on par with the documentation.

Reply:
The ExperimentId is generated in `packages/engine/bin/cli/src/manifest.rs:52` - `create_experiment_run_id` actually would imply it needs renaming. But basically the ExperimentId is the ExperimentName + some rng thing that gives it a unique identifier. 

However upon more thought, I don't know why we differentiate between ExperimentId and ExperimentRunId in the first place. The ExperimentId is unique per run it seems. That's maybe why the naming wasn't consistent in the first place. It's possible we wanted the ExperimentRunId (previously ExperimentId) to be a fixed length and known format for easier passing in flatbuffers and such, going to need to think on this

Reply:
Right, after following up on Slack we've decided this needs more thought, and will revert the rename and follow up with a new task later on